fdbserver/workloads/ConfigIncrement.actor.cpp

```diff
@@ -145,10 +145,7 @@ class ConfigIncrementWorkload : public TestWorkload {
 		auto localIncrementActors =
 		    (clientId < incrementActors) ? ((incrementActors - clientId - 1) / clientCount + 1) : 0;
 		for (int i = 0; i < localIncrementActors; ++i) {
-			// TODO: The timeout is a hack to get the test to pass before rollforward and
-			// rollback are supported. Eventually, this timeout should be removed so
-			// we test that all clients make progress.
-			actors.push_back(timeout(incrementActor(this, cx), 60.0, Void()));
+			actors.push_back(incrementActor(this, cx));
```

Comment:
After removing the timeout, there are occasional timeouts in the `ConfigIncrement` simulation test. They seem to occur because both `incrementActor`s have similar backoffs (even with the added randomness). Both clients end up in a back and forth trying to get their generation accepted by a majority of `ConfigNode`s, until eventually the delay between attempts becomes so large that the test just times out (the delay includes the multiplier `1 << numRetries`, which grows very quickly).

Reply:
Resolved offline. Clamped the value to a maximum wait (pre-jitter) of 20 seconds.